rasa/shared/utils/common.py

```diff
@@ -5,32 +5,59 @@
 import logging
 from typing import Text, Dict, Optional, Any, List, Callable, Collection
 
+import rasa.shared.utils.io as io_utils
+from rasa.shared.constants import NEXT_MAJOR_VERSION_FOR_DEPRECATIONS
+
+
 logger = logging.getLogger(__name__)
 
 
 def class_from_module_path(
-    module_path: Text, lookup_path: Optional[Text] = None
+    module_path: Text, lookup_path: Optional[Text] = None, ensure_class: bool = True
 ) -> Any:
     """Given the module name and path of a class, tries to retrieve the class.
 
-    The loaded class can be used to instantiate new objects."""
-    # load the module, will raise ImportError if module cannot be loaded
+    The loaded class can be used to instantiate new objects.
+
+    Args:
+        module_path: either an absolute path to a Python class,
+                     or the name of the class in the local / global scope.
+        lookup_path: a path where to load the class from, if it cannot
+                     be found in the local / global scope.
+        ensure_class: a boolean that will ensure that the loaded Python class
+                      is actually class if True (by default).
```

Comment:
In my opinion it's clear from the method signature that it's the default. Keeping information about the default in two places seems error prone to me.
```suggestion
                      is actually class if `True`.
```